streams/src/main/java/org/apache/kafka/streams/kstream/KStreamBuilder.java

```diff
@@ -55,7 +55,22 @@ public KStreamBuilder() {
      * @return a {@link KStream} for the specified topics
      */
     public <K, V> KStream<K, V> stream(String... topics) {
-        return stream(null, null, topics);
+        return stream(null, null, null, topics);
+    }
+
+
+    /**
+     * Create a {@link KStream} instance from the specified topics.
+     * The default deserializers specified in the config are used.
+     * <p>
+     * If multiple topics are specified there are nor ordering guaranteed for records from different topics.
+     *
+     * @param offsetReset the offset reset to use for this stream if no committed offsets available; acceptable values are earliest or latest
```

Comment:
nit: rename `offset reset` to `auto offset reset policy` in all occurrences in Javadocs.